content/docs/dvclive/usage.md

```diff
@@ -0,0 +1,144 @@
+# Usage
+
+We will use sample [MNIST classification](http://yann.lecun.com/exdb/mnist/)
+training code in order to see how one can introduce Dvclive into the workflow.
+
+> Note that [keras](https://keras.io/about/#installation-amp-compatibility) is
+> required throughout these examples.
+
+```python
+# train.py
+
+from keras.datasets import mnist
```

Comment:
Should this be hidden under a `<details>` section? I think I'm going to use that in **Dvclive with DVC** but maybe it's OK to have it expanded here, up to you.

Reply:
Nvmd about `<details>`. Resolving